storage/replica.go

```diff
@@ -1433,12 +1436,20 @@ func (r *Replica) handleRaftReady() error {
 func (r *Replica) tick() error {
 	r.mu.Lock()
 	defer r.mu.Unlock()
+	r.mu.ticks++
 	r.mu.raftGroup.Tick()
-	// TODO(tamird/bdarnell): Reproposals should occur less frequently than
-	// ticks, but this is acceptable for now.
-	// TODO(tamird/bdarnell): Add unit tests.
-	err := r.reproposePendingCmdsLocked()
-	return err
```

Comment:
any reason to keep incrementing vs resetting when larger than `RaftElectionTimeoutTicks`?